.golangci.yml

```diff
@@ -59,4 +59,4 @@ linters:
     - nonamedreturns
 
 issues:
-  new-from-rev: 3221f76
\ No newline at end of file
+  new-from-rev: 9af4477
```

Comment:
Had way to many lint problems, need to limit the scope of the linter to decrease linting time in a different pr